api/v2/api.go

```diff
@@ -220,11 +220,11 @@ func (api *API) getReceiversHandler(params receiver_ops.GetReceiversParams) midd
 			iname := integration.String()
 			sendResolved := integration.SendResolved()
 			integrations = append(integrations, &open_api_models.Integration{
-				Name:               &iname,
-				SendResolved:       &sendResolved,
-				LastNotify:         strfmt.DateTime(notify.UTC()),
-				LastNotifyDuration: duration.String(),
-				LastError: func() string {
+				Name:                &iname,
+				SendResolved:        &sendResolved,
+				LastAttempt:         strfmt.DateTime(notify.UTC()),
+				LastAttemptDuration: duration.String(),
+				LastAttemptError: func() string {
```

Comment:
```suggestion
				LastNotifyAttempt:         strfmt.DateTime(notify.UTC()),
				LastNotifyAttemptDuration: duration.String(),
				LastNotifyAttemptError: func() string {
```

if we do decide to go this route - I think we should prefix with notify to signal _what_ is the thing we're attempting to do.